frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketsTopPanel/IssuePropertiesInputs/issuePropertiesInputs.component.tsx

```diff
@@ -23,95 +23,105 @@ import { InputController } from '@controls/inputs/inputController.component';
-					onBlur={onBlur}
-					key={IssueProperties.PRIORITY}
-					values={PRIORITY_LEVELS_MAP}
-					disabled={readOnly}
-				/>
-			</PriorityInput>
-			<DueDateInput>
-				<PropertyTitle>
-					<FormattedMessage
-						id="ticketTopPanel.dueDate.label"
-						defaultMessage="Due"
+export const IssuePropertiesInputs = ({ onBlur, readOnly, properties }: IIssuePropertiesInputs) => {
+	const hasIssueProperties = properties.some((property) => property.name === IssueProperties.PRIORITY);
+
+	return (
+		<>
+			<FlexContainer>
+				<StatusInput>
+					<PropertyTitle>
+						<FormattedMessage
+							id="ticketTopPanel.status.label"
+							defaultMessage="Status"
+						/>
+					</PropertyTitle>
+					<FormChipSelect
+						variant="text"
+						tooltip={formatMessage({
+							id: 'customTicket.topPanel.status.tooltip',
+							defaultMessage: 'Set status',
+						})}
+						name={`properties.${BaseProperties.STATUS}`}
+						onBlur={onBlur}
+						key={BaseProperties.STATUS}
+						values={STATUS_MAP}
+						disabled={readOnly}
 					/>
-				</PropertyTitle>
-				<FormDueDateWithIcon
-					tooltip={formatMessage({
-						id: 'customTicket.topPanel.dueDate.tooltip',
-						defaultMessage: 'Set due date',
-					})}
-					name={`properties[${IssueProperties.DUE_DATE}]`}
-					onBlur={onBlur}
-					key={IssueProperties.DUE_DATE}
-					disabled={readOnly}
-				/>
-			</DueDateInput>
-			<StatusInput>
-				<PropertyTitle>
+				</StatusInput>
+				{hasIssueProperties && (
+					<>
+						<PriorityInput>
+							<PropertyTitle>
+								<FormattedMessage
+									id="ticketTopPanel.priority.label"
+									defaultMessage="Priority"
+								/>
+							</PropertyTitle>
+							<FormChipSelect
+								variant="text"
+								tooltip={formatMessage({
+									id: 'customTicket.topPanel.priority.tooltip',
+									defaultMessage: 'Set priority',
+								})}
+								name={`properties[${IssueProperties.PRIORITY}]`}
+								onBlur={onBlur}
+								key={IssueProperties.PRIORITY}
+								values={PRIORITY_LEVELS_MAP}
+								disabled={readOnly}
+							/>
+						</PriorityInput>
+						<DueDateInput>
+							<PropertyTitle>
+								<FormattedMessage
+									id="ticketTopPanel.dueDate.label"
+									defaultMessage="Due"
+								/>
+							</PropertyTitle>
+							<FormDueDateWithIcon
+								tooltip={formatMessage({
+									id: 'customTicket.topPanel.dueDate.tooltip',
+									defaultMessage: 'Set due date',
+								})}
+								name={`properties[${IssueProperties.DUE_DATE}]`}
+								onBlur={onBlur}
+								key={IssueProperties.DUE_DATE}
+								disabled={readOnly}
+							/>
+						</DueDateInput>
+					</>
+				)}
+			</FlexContainer>
+			<Gap $height="10px" />
+			<AssigneesContainer>
+				<Typography variant="h5">
 					<FormattedMessage
-						id="ticketTopPanel.status.label"
-						defaultMessage="Status"
+						id="customTicket.topPanel.assignees.label"
+						defaultMessage="Assignees"
```

Comment:
Should Assignees not be in the hasIssueProperties condition?

Reply:
No, because this PR changes the concept of status being a issue properties only into being a property for all tickets.

Reply:
Yes, but this isn't status it's assignees. This causes errors when I try to update the assignee value in the tickets table